core/google/cloud/streaming/http_wrapper.py

```
@@ -90,7 +90,7 @@ def _httplib2_debug_level(http_request, level, http=None):
     old_level = httplib2.debuglevel
     http_levels = {}
     httplib2.debuglevel = level
-    if http is not None:
+    if http is not None and hasattr(http, 'connections'):
```

`hasattr` is bad, it swallows exceptions, something something blah blah, use

```
getattr(http, 'connections', None) is not None
```